JS-Algorithms/challenges.js

```diff
@@ -95,7 +106,15 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+   let factorial = n;
+   let arrayFactorial = [];
+   let sum = 0;
+   for (let i = factorial - 1; i >= 1; i--) {
+      factorial *= i;
+   }
+   arrayFactorial = [...BigInt(factorial).toString()];
```

Comment:
Is this spread operator necessary?

Reply:
Yes because I'm creating an array of the digits of the factorial value using the spread operator 
//factorial = 123456789
arrayFactorial = [...BigInt(factorial).toString()];
//arrayFactorial = [1,2,3,4,5,6,7,8.9]
and also I use the BigInt parsing because the scientific notation